resources/server/players/player.controller.ts

```diff
@@ -65,18 +65,18 @@ if (!config.general.enableMultiChar) {
 // instantiating/deleting a player. The config option must be set to true
 // for these to be available
 if (config.general.enableMultiChar) {
-  // This has to be an event as FXServer does not yet support exports which return a promise.
-  on('npwd:newPlayer', async (playerDTO: PlayerAddData) => {
+  const exp = global.exports;
+
+  exp('newPlayer', async (playerDTO: PlayerAddData) => {
```

Comment:
Yay 🥳 I remember not doing these as exports because of our dear friend cfx_asyncRetVal. Appreciate the update.